components/mach_vendor.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, config):
 
     @logEntryExit
     def check_for_update(self, library):
-        parts = self.run(["./mach", "vendor", "--check-for-update", library.yaml_path]).stdout.decode().strip().split()
+        parts = self.run(["./mach", "vendor", "--check-for-update", library.yaml_path]).stdout.decode().strip().split(" ")
```

Comment:
Output is `revision date\n`. Because I strip() the newline from the output before calling split, split was operating on spaces. This just makes it explicit.